src/Types.d.ts

```diff
@@ -155,7 +155,7 @@ type StripeObject = {
   _requestSender: RequestSender;
   _getPropsFromConfig: (config: Record<string, unknown>) => UserProvidedConfig;
   _clientId?: string;
-  _platformFunctions: import('./platform/PlatformFunctions');
+  _platformFunctions: import('./platform/PlatformFunctions').PlatformFunctions;
```

Comment:
Should we do a real import on top?

Reply:
Moving these to the top [turns this into a local module s](https://stackoverflow.com/a/51114250) so we'd have to export / import types where they're needed (which might be what we want because it'll be easier to eventually get rid of this file later). I think that's a bigger lift than should fit in this PR, but if you think that's a good idea I'm happy to log this as a follow-up!

Reply:
Fine to do it later. We do need to clean up this file. Ideally it should go away completely.